packages/nextjs/app/builders/0x8488B81757074B0558ebB05D59A81F43F55bf0C8/page.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
```

Comment:
You don't need to use "use client" here, as this can be server-rendered. There are no client-side actions like useEffect or useState involved.